hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/ApproveAllowanceLogic.java

```diff
@@ -140,16 +142,19 @@ private void applyFungibleTokenAllowances(final List<TokenAllowance> tokenAllowa
             final var tokensMap = accountToApprove.getMutableFungibleTokenAllowances();
 
             final var spender = Id.fromGrpcAccount(allowance.getSpender());
-            accountStore.loadAccountOrFailWith(spender, INVALID_ALLOWANCE_SPENDER_ID);
-
             final var amount = allowance.getAmount();
             final var tokenId = allowance.getTokenId();
 
             final var key = FcTokenAllowanceId.from(EntityNum.fromTokenId(tokenId), spender.asEntityNum());
             if (tokensMap.containsKey(key) && amount == 0) {
+                // spender need to be validated as being a valid account when removing allowances,
```

Comment:
Is this also needed here?
`// spender need NOT be validated...`

Reply:
yes fixed.